plugins/wazo-gigaset/N510/templates/base.tpl

```diff
@@ -8,7 +8,7 @@
     <MAC_ADDRESS value="{{ XX_mac_addr }}"/>
     <PROFILE_NAME class="string" value="N510"/>
     {%- if http_port %}
-    <S_SPECIAL_DATA_SRV class="string" value='"http://{{ ip }}:{{ http_port }}/Gigaset/merkur258_42.bin?mac={{ XX_mac_addr }}&merkur258_42.bin"'/>
+    <S_SPECIAL_DATA_SRV class="string" value='"http://{{ ip }}:{{ http_port }}/42/fw262.bin?mac={{ XX_mac_addr }}&fw262.bin"'/>
```

Comment:
@fouille could you please check this point?

Reply:
@bloom1 I respect the naming convention from Gigaset documentation : 
```
Important is to use the naming like described in the example: "merkur"softwareversion"_42.bin"
``` 
I choose the directory `/Gigaset` because the file `merkur"softwareversion"_42.bin` is in this directory, for respect standard implementation of Gigaset tree structure.